cli/users.local.yml

```diff
@@ -7,6 +7,12 @@ environment:
   - AUTHENTICATE_VIA_JUPYTER=
   - WORKSPACE_BASE_URL=${username}
 shm_size: 512m
+deploy:
```

Comment:
Also move shm_size into the resources. Convert the constant value of 512m into a variable value loaded from `dtaas.toml`. Update the code to satisfy this requirement.